bot/exts/code_jams/_cog.py

```diff
@@ -16,7 +16,8 @@
 from bot.exts.code_jams import _creation_utils
 from bot.exts.code_jams._flows import (creation_flow, deletion_flow, move_flow,
                                        remove_flow)
-from bot.exts.code_jams._views import JamConfirmation, JamTeamInfoConfirmation
+from bot.exts.code_jams._views import (JamConfirmation, JamInfoView,
+                                       JamTeamInfoConfirmation)
```

Comment:
This goes against our style guidelines, and it's inconsistent with other code in this bot. In fact, it's inconsistent with other code in this PR's diff.

Reply:
Poorly formatted, must have accidentally formatted it with PyCharm. I'll patch it immediately.